src/policy.c

```diff
@@ -1299,6 +1326,7 @@ policy_filter_unwanted(Solver *solv, Queue *plist, int mode)
 	  return;
 	}
     }
+  prune_to_considered(pool, plist);
```

Comment:
this call should be guarded by `if plist->count > 1`

Reply:
The test is part of the function